packages/wrangler/src/cloudchamber/build.ts

```diff
@@ -126,18 +126,38 @@ export async function buildAndMaybePush(
 			account: account,
 			containerApp: containerConfig,
 		});
-
+		let pushed = false;
 		if (push) {
 			await dockerLoginManagedRegistry(pathToDocker);
 			try {
+				// we don't try to parse repoDigests until here because we
+				// don't want to fail on parse errors if we aren't pushing anyways
+				// as it's only relevant when pushing.
+				const parsedDigests = JSON.parse(repoDigests);
```

Comment:
are `repoDigests` the same as Docker image digests?

```
A Docker image digest is a unique, cryptographic identifier (SHA-256 hash) representing the content of a Docker image. Unlike tags, which can be reused or changed, a digest is immutable and ensures that the exact same image is pulled every time. This guarantees consistency across different environments and deployments.
```
https://docs.docker.com/dhi/core-concepts/digests/